### PR TITLE
Create chemical-and-pharmaceutical-bulletin.csl

### PR DESCRIPTION
**Pull request by shojitakahashi**

### CODE REVIEW

File: chemical-and-pharmaceutical-bulletin.csl [hunk 1]
```diff
@@ -0,0 +1,232 @@
...
+	<title-short>Chem. Pharm. Bull.</title-short>
+    <id>http://www.zotero.org/styles/chemical-and-pharmaceutical-bulletin</id>
+    <link href="http://www.zotero.org/styles/chemical-and-pharmaceutical-bulletin" rel="self"/>
+    <link href="http://cpb.pharm.or.jp/document/inst_e.pdf" rel="documentation"/>
+    <author>
+      <name>Shoji Takahashi</name>
+      <email>s.takahashi@elsevier.com</email>
+      <uri>http://www.mendeley.com/profiles/shoji-takahashi3/</uri>
+    </author>
+    <category citation-format="numeric"/>
```
Comment (adam3smith) [inline chemical-and-pharmaceutical-bulletin.csl]: could you add `<category field="chemistry"/>` here, please?

**Comment by adam3smith** [comment]
that's it. Looking good otherwise.

File: chemical-and-pharmaceutical-bulletin.csl [hunk 2]
```diff
@@ -0,0 +1,232 @@
...
+  </macro>
+  <macro name="event">
+    <group delimiter=", ">
+      <text variable="event-place"/>
+      <group>
+        <text value="on " suffix=" "/>
+        <date variable="event-date">
+          <date-part name="day" range-delimiter="" suffix=" "/>
+          <date-part name="month" range-delimiter="" suffix=", "/>
+          <date-part name="year" range-delimiter=""/>
```
Comment (adam3smith) [inline chemical-and-pharmaceutical-bulletin.csl]: same issue with range delimiters

File: chemical-and-pharmaceutical-bulletin.csl [hunk 3]
```diff
@@ -0,0 +1,232 @@
...
+  </macro>
+  <macro name="edition">
+    <number suffix=" " variable="edition" form="ordinal"/>
+    <label plural="never" variable="edition" form="short"/>
+  </macro>
+  <macro name="event">
+    <group delimiter=", ">
+      <text variable="event-place"/>
+      <group>
+        <text value="on " suffix=" "/>
```
Comment (adam3smith) [inline chemical-and-pharmaceutical-bulletin.csl]: duplicate space.

**Comment by rmzelle** [comment]
Thanks!

**Pull request merged**